packages/react-components/react-toolbar/stories/Toolbar/index.stories.tsx

```diff
@@ -19,7 +19,7 @@ export { VerticalButton } from './ToolbarVerticalButton.stories';
 export { FarGroup } from './ToolbarFarGroup.stories';
 
 export default {
-  title: 'Preview Components/Toolbar',
+  title: 'Toolbar',
   component: Toolbar,
```

Comment:
Any plan to add props tables for `Toolbar` children components?

Reply:
Is done in #26382